test/helper.rb

```diff
@@ -1,3 +1,9 @@
+require "simplecov"
+SimpleCov.start do
+  add_filter "/test/"
+  add_filter "/vendor/"
```

Comment:
I think the concern is that on CI, it installs the bundled gems to `./vendor` and we don't want those to impact the coverage percentage.

I mostly copied this from other projects, so I'm not certain, and now I'm curious... so I'll try pushing without this line and check the CI output to see if the coverage goes down.